netlify.toml

```diff
@@ -1,2 +1,3 @@
 [build]
-  base = "packages/edge-demo-site"
\ No newline at end of file
+command = "npm run build:edge-demo"
```

Comment:
@ascorbic, all the files in [`e9927b0` (#16)](https://github.com/netlify/remix-compute/pull/16/commits/e9927b0f097f9f9a1f1067d760da2aab260fae3e) is the best setup for a deploy I could get going. Also, the change in https://github.com/netlify/remix-compute/commit/4333b20ce5821b19478ea176db07e969a8ec1aa6#diff-07374dfaf56a1e11c7f1b9216123fb94be5458e2ca11f70f170fdc9a283aee72R9.

It's a monorepo where the edge demo site requires the `@netlify/remix-edge-adapter` and `@netlify/remix-runtime` to be built first so that the edge demo site can build. I tried the `cd` trick with a base URL in the root netlify.toml, but couldn't get that working as the base url of `packages/edge-demo-site` wasn't installing all the packages for the monorepo.

That was also why I had issues with `if-env` and `husky` not being found on git push deploys. Definitely open to suggestions as I imagine at some point soon, the Netlify Functions demo site will be in here as well.

Reply:
Rather than the toml you could set these in the UI, which would let you use different commands and roots for each site

Reply:
Yeah, good call about using different commands. The root would stay the same though as I need the monorepo to install everything. I'll make this change once we bring the Netlify Functions adapter into the project.